### DISCUSSION TITLE
Problem with Interaction.s.sol - Getting a  vm.keyExistsJson error

### DISCUSSION THREAD

**Discussion by Hardeybola** [Cyfrin/foundry-full-course-cu#1756] [Q&A]
Hello guys, how are you all doing. 
I'm having a problem writing interactions.s.sol.
In Patricks course, we re supposed to install Foundry Dev ops so we can get our most recent  deployed contract.

I installed it and after writing my code, when im run it, i get an error directing me to DevOpsTools saying

"Member "keyExistsJson" not found or not visible after argument-dependent lookup in contract Vm"

i dont know if thos error is common and what the solution is. thank you.

![Screenshot 2024-05-27 223314](https://github.com/Cyfrin/foundry-full-course-f23/assets/110025630/092a15f0-1827-4009-8454-d575302c284d)

**Comment by EngrPips** [comment]
Hello @Hardeybola, I don't seem to have that issue on my end; please delete that `foundry-devops` and install a new one using `forge install Cyfrin/foundry-devops --no-commit` then continue whatever you are doing and let's see if you still run into the same problem.

**Reply by Hardeybola** [reply]
Hello @EngrPips  i did what you suggested, but im still running into the same problem

**Reply by EngrPips** [reply]
That's weird, I am not running into this issue on my end.
